.ruby-version

```diff
@@ -1 +1 @@
-2.0.0-p247
+2.0.0-p645
```

Comment:
This only affects the version of Ruby used locally: in production, it uses the system Ruby package, which is 2.0.0-something.